drush/Commands/DeployCommands.php

```diff
@@ -91,7 +91,7 @@ public function backstop(string $target, string $reference, array $options = ['c
       throw new \Exception('CircleCI API response was a ' . $code . '. Use -v for more Guzzle information.');
     }
 
-    // $body = json_decode((string)$response->getBody(), TRUE);
+    $body = json_decode((string)$response->getBody(), TRUE);
```

Comment:
We were getting a warning about $body being undefined. If there's still a good reason to keep this commented out, feel free to comment it out again.